modules/developer_manual/pages/general/security.adoc

```diff
@@ -244,7 +244,7 @@ in unprotected requests.
 link:https://www.owasp.org/index.php/HTTP_Response_Splitting[HTTP Response Splitting], 
 check all request variables for `%0d` (CR) and `%0a` (LF),
 if they are parameters provided to
-link: http://php.net/manual/en/function.header.php[PHP’s header() function].
+link:http://php.net/manual/en/function.header.php[PHP’s header() function].
```

Comment:
Please remove the `link` prefix and change the special single quote to a standard, ascii, quote, i.e., `PHP’s` -> `PHP's`.